scripts/test/SANS/gui_logic/state_gui_model_test.py

```diff
@@ -299,14 +299,14 @@ def test_that_transmission_monitor_can_be_set(self):
         state_gui_model.transmission_monitor = 4
         self.assertTrue(state_gui_model.transmission_monitor == 4)
 
-    def test_that_transmission_m4_shift_default_is_empty(self):
+    def test_that_transmission_mn_shift_default_is_empty(self):
         state_gui_model = StateGuiModel({"test": [1]})
-        self.assertTrue(state_gui_model.transmission_m4_shift == "")
+        self.assertTrue(state_gui_model.transmission_mn_shift == "")
 
-    def test_that_transmission_m4_shift_can_be_set(self):
+    def test_that_transmission_mn_shift_can_be_set(self):
         state_gui_model = StateGuiModel({"test": [1]})
-        state_gui_model.transmission_m4_shift = 234
-        self.assertTrue(state_gui_model.transmission_m4_shift == 234)
+        state_gui_model.transmission_mn_shift = 234
+        self.assertTrue(state_gui_model.transmission_mn_shift == 234)
```

Comment:
why 234?

Reply:
I am setting the number on the `state_gui_model` object and ensuring that it has been set here. `234` is a just a number for testing.

Reply:
is the number likely to ever change? Im just wondering if it may be worth assigning it to a variable, so if it does change then only one number has to be edited.

Reply:
The number itself is irrelevant. This is a test, no variables needed.

Reply:
ok.